pkg/shootflavors/extendedflavors_test.go

```diff
@@ -82,9 +82,11 @@ var _ = Describe("extended flavor test", func() {
 	})
 
 	It("should return 1 shoot", func() {
+		true := true
 		rawFlavors := []*common.ExtendedShootFlavor{{
 			ExtendedConfiguration: defaultExtendedCfg,
 			ShootFlavor: common.ShootFlavor{
+				AllowPrivilegedContainers: &true,
```

Comment:
nit: you can just use the k8s pointer package `pointer.BoolPtr(true)`

Reply:
done